internal/impl/devbox.go

```diff
@@ -682,6 +681,47 @@ func (d *Devbox) printPackageUpdateMessage(
 	return nil
 }
 
+func (d *Devbox) computeNixEnv() ([]string, error) {
+	nixShellFilePath := filepath.Join(d.projectDir, ".devbox/gen/shell.nix")
+	vaf, err := nix.PrintDevEnv(nixShellFilePath)
```

Comment:
use: `d.nixShellFilePath()` but first please approve https://github.com/jetpack-io/devbox/pull/584 so it can be landed :)

Reply:
oops, sorry for not seeing that earlier!  merging in main and updating...